webapp/src/features/menu/DisplayRsuErrors.tsx

```diff
@@ -0,0 +1,265 @@
+              <ThemeProvider theme={theme}>
+                <Accordion>
+                  <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                    <Typography>Online Status</Typography>
+                  </AccordionSummary>
+                  <AccordionDetails>
+                    <div style={errorPageStyle}>
+                      <p>
+                        <b>RSU Online Status: </b>
+                        {getRSUOnlineStatus(selectedRSU.properties.ipv4_address)}
+                      </p>
+                      <br />
+                      <p>
+                        <b>RSU Last Online: </b>
+                        {getRSULastOnline(selectedRSU.properties.ipv4_address)}
+                      </p>
+                    </div>
+                  </AccordionDetails>
+                </Accordion>
+                <Accordion>
+                  <AccordionSummary expandIcon={<ExpandMoreIcon />}>
+                    <Typography>SCMS Status</Typography>
+                  </AccordionSummary>
+                  <AccordionDetails>
+                    <div style={errorPageStyle}>
+                      <p>
+                        <b>SCMS Status: </b>
+                        {getRSUSCMSStatus(selectedRSU.properties.ipv4_address) === '1' ? 'Healthy' : 'Unhealthy'}
+                      </p>
+                      <br />
+                      <p>
+                        <b>SCMS Expiration: </b>
+                        {getRSUSCMSExpiration(selectedRSU.properties.ipv4_address)}
+                      </p>
+                    </div>
+                  </AccordionDetails>
+                </Accordion>
+              </ThemeProvider>
+            </StyledEngineProvider>
+          </div>
+          <div style={{ display: 'flex', alignContent: 'center', justifyContent: 'center', flexDirection: 'row' }}>
+            <Button
+              variant="contained"
+              style={{ margin: '1rem', backgroundColor: '#b55e12', color: 'white' }}
+              onClick={() => {
+                setEmailHidden(false)
+              }}
+            >
+              Generate Error Summary Email
+            </Button>
+          </div>
+          <RsuErrorSummary
+            rsu={selectedRSU.properties.ipv4_address}
+            online_status={
+              getRSUOnlineStatus(selectedRSU.properties.ipv4_address) +
+              (getRSULastOnline(selectedRSU.properties.ipv4_address) === 'No Data'
+                ? ' (Never Online)'
+                : ' (Last Online ' + getRSULastOnline(selectedRSU.properties.ipv4_address) + ')')
+            }
+            scms_status={getRSUSCMSDisplay(selectedRSU.properties.ipv4_address)}
+            hidden={emailHidden}
+            setHidden={setHidden}
+          />
+        </div>
+      ) : (
+        <div id="container" className="sideBarOn" style={{ width: '95%' }}>
+          <h1 className="h1" style={{ marginBottom: '1rem', width: '100%', textAlign: 'center' }}>
+            RSU Errors
```

Comment:
Is this table only displaying RSUs with errors? If not, maybe it makes more sense to call this RSU Status? I think the RSU navigation element is super useful for even working RSUs. You could look at changing the status text color for each RSU so it is visually easier to spot issues too.

Reply:
Updated the table name to RSU Status and set text color to green/red depending on error status